src/fusion_power_plant.cc

```diff
@@ -34,28 +34,32 @@ std::string FusionPowerPlant::str() {
   return Facility::str();
 }
 
+
+
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 void FusionPowerPlant::EnterNotify() {
   cyclus::Facility::EnterNotify();
 
   fuel_usage_mass = (burn_rate * (fusion_power / MW_to_GW) / 
     (kDefaultTimeStepDur * 12) * context()->dt());
-  //fuel_usage_atoms = fuel_usage_mass / tritium_atomic_mass;
   blanket_turnover = blanket_size * blanket_turnover_fraction;
-
+  double startup_inventory = reserve_inventory + sequestered_equilibrium; 
+  
   //Create the blanket material for use in the core, no idea if this works...
-  blanket = Material::Create(this, 0.0, 
+   blanket = Material::Create(this, 0.0, 
```

Comment:
wrong indentation - may use something like `clang-format` to ensure consistent formatting.